apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php

```diff
@@ -204,10 +204,12 @@ public function output_template_content( $template_type ) {
 		// 10 priority
 		$content = wptexturize( $content );
 		// @todo maybe look at WPCOM_Responsive_Images for WPCom responsive image handling
-
 		// 11 priority
 		$content = do_shortcode( $content );
 
+		$content = prepend_attachment( $content );
+		$content = wp_make_content_images_responsive( $content );
```

Comment:
Non-blocking, but I think we'll probably want to start tracking these as units in the isolated suite on the wpcom side. It makes it much easier to visualize the post_content transform and catch regressions.

Reply:
Feel free to DM if you need help with test setup.